jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslContextFactory.java

```diff
@@ -1131,6 +1131,9 @@ public SSLContext getSslContext()
 
         synchronized (this)
         {
+            if (_factory == null)
+                throw new IllegalStateException("reload failed SslContext unavailable");
```

Comment:
Change the message to "SslContextFactory not started or reload failed" in the 3 places you have added it.

We will know from the stack trace what is unavailable.

Reply:
There is this check just above the synchronized block
```java
if (!isStarted())
    return _setContext;
```
which is why I didn't put the "not started" bit in the exception message, or do you expect this state to be able to change in between this check and the synchronized block?

Reply:
Cool, just say "SslContextFactory reload failed" then.